keylime/cloud_verifier_tornado.py

```diff
@@ -848,10 +918,11 @@ def main():
     logger.info('Starting Cloud Verifier (tornado) on port %s, use <Ctrl-C> to stop', cloudverifier_port)
 
     app = tornado.web.Application([
-        (r"/(?:v[0-9]/)?agents/.*", AgentsHandler),
-        (r"/(?:v[0-9]/)?allowlists/.*", AllowlistHandler),
+        (r"/v?[0-9]+(?:\.[0-9]+)?/agents/.*", AgentsHandler),
+        (r"/v?[0-9]+(?:\.[0-9]+)?/allowlists/.*", AllowlistHandler),
+        (r"/versions?", VersionHandler),
         (r".*", MainHandler),
-    ])
+    ], debug=True)
```

Comment:
shouldn't this be selectively set when the `DEBUG` verbosity is specified?

Reply:
Just removing it instead since it's not really related to this change. If we do decide to do that we should enable it across everything.